Service/EventBridgeNotifier.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Aligent\EventBridge\Service;
+
+use Aligent\EventBridge\Model\Config as EventBridgeConfig;
+use Aligent\Webhooks\Service\Webhook\NotifierInterface;
+use Aligent\Webhooks\Api\Data\WebhookInterface;
+use Aligent\Webhooks\Helper\NotifierResult;
+use Magento\Framework\Encryption\EncryptorInterface;
+use Magento\Framework\Serialize\Serializer\Json;
+use Aws\EventBridge\EventBridgeClient;
+use Psr\Log\LoggerInterface;
+
+/**
+ * Class EventBridgeNotifier
+ *
+ * A notifier for relaying events into AWS EventBridge.
+ *
+ */
+class EventBridgeNotifier implements NotifierInterface
+{
+    /**
+     * @var Json
+     */
+    private Json $json;
+
+    /**
+     * @var EncryptorInterface
+     */
+    private EncryptorInterface $encryptor;
+
+    /**
+     * @var LoggerInterface
+     */
+    private LoggerInterface $logger;
+
+    /**
+     * @var EventBridgeConfig
+     */
+    private EventBridgeConfig $config;
+
+    /**
+     * @var EventBridgeClient
+     */
+    private $eventBridgeClient;
+
+    public function __construct(
+        Json $json,
+        EncryptorInterface $encryptor,
+        LoggerInterface $logger,
+        EventBridgeConfig $config
+    ) {
+        $this->json = $json;
+        $this->encryptor = $encryptor;
+        $this->logger = $logger;
+        $this->config = $config;
+
+        $this->eventBridgeClient = new EventBridgeClient([
+            'version' => '2015-10-07',
+            'region' => $this->config->getAWSRegion(),
+            'credentials' => [
+                'key' => $this->config->getAWSKeyId(),
+                'secret' => $this->encryptor->decrypt($this->config->getAWSSecretKey())
+            ]
+        ]);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function notify(WebhookInterface $webhook, array $data): NotifierResult
+    {
+        $notifierResult = new NotifierResult();
+        $notifierResult->setSubscriptionId($webhook->getSubscriptionId());
+
+        try {
+            $eventEntry['EventBusName'] = $this->config->getEventBridgeBus();
+            $result = $this->eventBridgeClient->putEvents([
```

Comment:
had a glance at the lib code for this but couldn't dig deeper. will the client throw an exception in case of an error or will it contain that error message within the `$result` object? if it does not throw an exception then you might want to cater for that and set `$notifierResult->setSuccess` appropriately as that will determine whether exponential retry to kick in